tests/testthat/test_base_measures.R

```diff
@@ -245,6 +245,11 @@ test_that("check measure calculations", {
   acc.perf = performance(pred.classif, measures = acc, model = mod.classif)
   expect_equal(acc.test, acc$fun(pred = pred.classif))
   expect_equal(acc.test, as.numeric(acc.perf))
+  #multiclass.brier
+  multiclass.brier.test = mean(rowSums((getPredictionProbabilities(pred.classif) - matrix(c(0,0,1,0,1,0,0,1,0,1,0,0), 4, 3))^2))
+  multiclass.brier.perf = performance(pred.classif, measures = multiclass.brier, model = mod.classif)
+  expect_equal(multiclass.brier.test, multiclass.brier$fun(pred = pred.classif))
+  expect_equal(multiclass.brier.test, as.numeric(multiclass.brier.test))
```

Comment:
Should read `expect_equal(multiclass.brier.test, as.numeric(multiclass.brier.perf))` no?